### ISSUE TITLE
When auditing a custom route, show in sidebar

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#3272]
When you're auditing a custom route, I suggest showing that in the sidebar here:
![image](https://github.com/ProjectSidewalk/SidewalkWebpage/assets/1621749/1e632168-12fb-4337-9bde-2a69dd9b129e)

Instead of:

> Current Mission
> Explore 1000 ft of this neighborhood

We have:

> Custom Route
> Finish the custom route (X.X mi)

Could have link to the custom route here so it would take you back to routebuilder or allow you to share it with others

**Comment by misaugstad** [comment]
I like this except for the part where we make the mission the full length of the route. We don't put a cap on the length of a route, so we should still probably split up long routes into smaller missions

**Comment by jonfroehlich** [comment]
Perhaps but IIRC, there is no way to reselect the route in the future, correct? So, if the user leaves the neighborhood (by starting a mission somewhere else) then no way to re-select this route...

**Comment by misaugstad** [comment]
Correct! I agree that we should add a way for users to be able to resume their routes if they go to a different neighborhood. But the suggestion looks like it would also turn the route into one really long mission. We just need to find a better way/location to show this info.

**Comment by jonfroehlich** [comment]
OK. How about just:

> Current Mission: Route X
> Explore 1000 ft of this route

Where Route X is hyperlinked and returns to the RouteBuilder so you can view it? <-- but if that's too complicated, just make the MVP the text above. Thoughts?

**Comment by misaugstad** [comment]
We don't have the ability to view routes again in RouteBuilder yet, but showing the route ID is easy enough!

**Comment by misaugstad** [comment]
I agree that I want info on where you are within your route, possibly part of a larger sidebar redesign..? :eyes: #2260 

Though I disagree that turning the custom route into one large mission is a good idea. Missions are generally very short, so we lose out on a lot of the gratification/gamification that users are supposed to experience.

I do think that we could be smarter about how long we make the missions, or making the route one mission if the route is small enough, for example.

But the change that will make all of this clearer is just to give a better idea of where users are in their overall route at all times.

**Comment by jonfroehlich** [comment]
> But the change that will make all of this clearer is just to give a better idea of where users are in their overall route at all times.
Totally agree.

Thanks for the discussion Mikey (both here and over Slack).

**Comment by misaugstad** [comment]
We now show a lot more about your route on the minimap!
